sdgym/summary.py

```diff
@@ -268,8 +270,13 @@ def _add_summary(data, modality, baselines, writer):
     add_sheet(performance, f'Performance ({modality})', writer, cell_fmt, index_fmt, header_fmt)
     add_sheet(error_summary, f'Errors Summary ({modality})', writer, cell_fmt, index_fmt,
               header_fmt)
```

Comment:
Small nitpick, but I think if the call below is being split out into multiple lines then this call should be as well.